src/V3File.cpp

```diff
@@ -700,6 +700,10 @@ int V3OutFormatter::endLevels(const char* strg) {
 }
 
 void V3OutFormatter::puts(const char* strg) {
+    if (!v3Global.opt.formatCxxCode()) {
+        for (const char* cp = strg; *cp; cp++) { putcOutput(*cp); }
```

Comment:
```suggestion
        for (const char* cp = strg; *cp; cp++) putcOutput(*cp);
```
But if you want speed I think you can just puts at this point, rather than doing it by character

Reply:
That's right. I've added extra function that can save whole string instead do it by character.